app/models/user.rb

```diff
@@ -105,6 +105,10 @@ def collect_analytics?
     Settings.analytics_enabled && !super_admin?
   end
 
+  def signed_in!
+    update(last_signed_in_at: Time.zone.now)
```

Comment:
Does this need to be `update!` or is there some magic that means this gets saved?

Reply:
Ah, good point, it should be `update!`. Both `update` and `update!` save the changes, the difference is that `update!` raises an exception if there is a failure in saving while `update` just returns false if there is an issue.

Reply:
(from https://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update)

Reply:
Fixed now

Reply:
Is there a test we can add to for this? Maybe `authentication_controller_spec`?